Presentation/PresentationStyle.swift

```diff
@@ -121,7 +121,7 @@ public extension PresentationStyle {
                 }
             }
 
-            bag += presented.installDismissButton().onValue {
+            bag += viewController.installDismissButton().onValue {
```

Comment:
I guess both need to be kept. In reality I don't think that both a navigation controller and the vc inside it will be configured with dismiss buttons but if so that would be a configuration error and both need to be shown. Don't think that Presentation should choose which one to show although we can check for which vc we have a `dismissBarItem`. 
```suggestion
            bag += viewController.installDismissButton().onValue {
                completion(.failure(PresentError.dismissed))
            }
            presented.installDismissButton().onValue {
```